lib/publish.js

```diff
@@ -14,10 +14,10 @@ module.exports = async (pluginConfig, {repositoryUrl}, {gitHead, gitTag, notes},
   debug('release ref: %o', gitHead);
 
   debug('Update git tag %o with commit %o and release description', gitTag, gitHead);
-  await got.put(urlJoin(apiUrl, `/projects/${repoId}/repository/tags/${gitTag}/release`), {
+  await got.post(urlJoin(apiUrl, `/projects/${repoId}/repository/tags/${gitTag}/release`), {
     json: true,
     headers: {'PRIVATE-TOKEN': gitlabToken},
-    body: {tag_name: gitTag, ref: gitHead, release_description: notes}, // eslint-disable-line camelcase
+    body: {id: repoId, tag_name: gitTag, description: notes}, // eslint-disable-line camelcase
```

Comment:
The `id` is not required in the payload, only in the URI (`/projects/${repoId}/repository/tags/${gitTag}/release`).
This line should be `body: {tag_name: gitTag, description: notes}`